.prettierignore

```diff
@@ -0,0 +1 @@
+*.d.ts
```

Comment:
We can bring this back, at least for our own d.ts files (seems fine to do in a follow up though)

Reply:
The d.ts files inside here are entirely build artifacts. The reason this failed in CI but not locally is that I had no run tsc before linting.

Reply:
gotcha